### PR TITLE
Improvements in the GitHub action summary report

### PR DESCRIPTION
**Pull request by gdelgadot**

### CODE REVIEW

**Comment by gdelgadot** [comment]
The GitHub Action summary report has been updated:

**Comment by gdelgadot** [comment]
Tests table
Screenshot 2023-10-19 at 16 15 40

File: src/action.ts [hunk 1]
```diff
@@ -406,22 +409,26 @@ class Action {
...
-        vulnerabilitiesTable += this.vulnerabilitiesTableRow(task.action_id, task.vulnerabilities.minimal, task.vulnerabilities.low, 
+        vulnerabilitiesTable += this.vulnerabilitiesTableRow(task.action_id, task.architecture || 'amd64',
+          task.vulnerabilities.minimal, task.vulnerabilities.low, 
           task.vulnerabilities.medium, task.vulnerabilities.high, task.vulnerabilities.critical, task.vulnerabilities.unknown, task.passed)
-        vulnerabilitiesTable += "<tr><td colspan=8>ℹ️ The CVE vulnerabilities are related to" 
-          + " the threshold and vulnerabilities types configured previously by the user.</td></tr>"  
       }
     }
 
+    if (!infoMessageDisplayed) {
```
[the remark this answers is not in the document]
Comment (gdelgadot) [reply src/action.ts]: Done

**Comment by gdelgadot** [comment]
After testing different ways to show the white circle for Skipped tests in the summary report and not finding any suitable option, @beltran-rubo and I have decided to leave the skipped tests column without emoticon.
Screenshot 2023-10-20 at 11 25 00

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "@actions/core": "^1.10.0",
     "@actions/glob": "^0.2.0",
     "@actions/io": "^1.1.1",
+    "@babel/core": "^7.23.2",
```
Comment (fevisera) [inline package.json:35]: btw, why is that module being required now?

**Comment by fevisera** [comment]
I left some comments you may like to consider, nevertheless LGTM!

**Review by fevisera — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (gdelgadot) [reply package.json]: This change is related to the [vulnerability alert](https://github.com/vmware-labs/vmware-image-builder-action/security/dependabot/15)

**Pull request merged**